src/big/big_float.cr

```diff
@@ -17,6 +18,23 @@ struct BigFloat < Float
     LibGMP.mpf_init_set_str(out @mpf, str, 10)
   end
 
+  def initialize(num : BigInt)
```

Comment:
@RX14 btw, these constructors also part of another #4653 pull request. They are cross in some areas.